setup.py

```diff
@@ -56,9 +56,10 @@ def alltests():
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
```

Comment:
Good catches. I see the last release claimed to remove support, but the 3.3 classifier got left behind and the 3.6 classifier wasn't added.

Reply:
I have a terrible Python script to verify the consistency of classifiers, travis.yml, tox.ini, appveyor.yml and manylinux build shell scripts: https://github.com/mgedmin/scripts/blob/master/check-python-versions

Maybe it could be cleaned up and hooked up to Travis CI?